modules/navier_stokes/test/tests/ins/boussinesq/tests

```diff
@@ -7,6 +7,14 @@
     exodiff = boussinesq_square_out.e
     requirement = 'The system shall be able to simulate natural convection by adding the Boussinesq approximation to the incompressible Navier-Stokes equations.'
   []
+  [threaded_exo]
+    type = Exodiff
+    input = boussinesq_square.i
+    exodiff = boussinesq_square_out.e
+    requirement = 'The system shall be able to solve mass, momentum, and energy incompressible Navier-Stokes equations with multiple threads.'
+    min_threads = 2
```

Comment:
I guess this is a short term solution before we add the multi-thread test target.

Reply:
Yes although it would obviously work still even after we added the threading target. We would just have duplicate tests when running that target.